docs/guide/tools/geometry.md

```diff
@@ -0,0 +1,159 @@
+# Geometry tools
+
+Xournal++ contains two geometry tools, a setsquare tool and a compass. They can be used for
+measuring lengths of elements in the document and as a guide for drawing straight lines,
+circles or parts thereof. You can even do geometric constructions with them. Both geometry
+tools can be manipulated via touch on a touchscreen or via the keyboard. Drawing happens
+through the mouse or stylus.
+
+## Setsquare overview
+
+With the setsquare you can measure distances and angles and draw straight lines as well as
+perpendicular lines to already drawn lines.
+
+![Setsquare-Overview](../../img/guide/setsquare_overview.png)
+
+The longest side of the setsquare is used for measuring elements of the document and drawing
+straight lines aligned to it.
+There are marks (in green) parallel to the longest side of the setsquare, that allow
+measuring distances from that side.
+
+Angular marks (in purple) help measuring angles. There is also a display of the current angle
+(in tenth of degrees) between the longest side of the setsquare and a horizontal line.
+The number always looks up. On the picture this angle is 10.0 degrees.
+
+The shorter two sides of the setsquare are not used for drawing lines aligned to them. Instead
+they are used for drawing lines to the middle point of the longest side of the setsquare.
+
+## Compass overview
+
+The compass serves for transporting distances and angles, for drawing circle arcs and sectors and
+for geometric constructions. The compass looks like a wheel.
+
+![Compass-Overview](../../img/guide/compass_overview.png)
+
+The outline of the compass is used for drawing circles, circle arcs and (when "fill" is enabled)
+circle sectors.
+
+There is a marked radius with measuring marks, along which lines can be drawn. The radius of the
+compass/drawn circles can be seen from the last displayed measuring mark.
+
+Angular marks (in purple) help measuring angles. Like for the setsquare there is a display of the
+current angle.
+
+## Units, distinguished point and axis
+
+For all geometry tools lenths are measured in centimeters and angles are measured in degrees.
+
+Each geometry tool has a distinguished point and a distinguished axis through that point.
+The distinguished point is used as a rotation and scaling center when rotating/scaling via keyboard.
+Angles are measured between horizontal axis and distinguished axis.
+
+- **Setsquare**: The distinguished axis and point are the longest side of the setsquare and its
+  midpoint, respectively.
+- **Compass**: Its distinguished point is the center of the compass wheel. The distinguished axis
+  is a radius that carries measuring marks and can be used to draw a radius or part of it.
+
+Using the keyboard (`m` key) the current distinguished point can be marked with an "x"-like shape
+that is added to the page. For example when drawing a circle the center using the compass,
+the center of the circle can be marked like that.
```

Comment:
These should be moved to the keyboard controls section. They seem out of place here.

Reply:
Since it is already written in the keyboard controls section, I have just removed it here.